js/src/scrollspy.js

```diff
@@ -212,6 +217,7 @@ const ScrollSpy = (($) => {
         if (this._activeTarget !== target) {
           this._activate(target)
         }
+        return
```

Comment:
Because it could happen that `scrollTop` is lower than `offset` of the last target despite it is the end of the page and it would overwrite `this._activeTarget` to the last but one target (line 235).